public_html/templates/leaderboard.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base.html" %}
+{% import _self as macros %}
+
+{# must pass in wiki and username as view-level variables are not accessible in macros #}
+{% macro leaderboard_list( wiki, dataset, username ) %}
+  <ol class="leaderboard-list">
+    {% for leader in dataset %}
+      <li class="ellipsis">
+        <a target="_blank" href="{{ wiki }}/wiki/User:{{ leader['user'] }}" class="{{ username == leader['user'] ? 'mine' : '' }}">{{ leader['user'] }}</a>
+        :
+        {{ leader['count'] }}</li>
+    {% endfor %}
+  </ol>
+{% endmacro %}
+
+{% block main %}
+  <section class="leaderboard container">
+    <a class="back-to-copypatrol" href="{{ siteUrl( '' ) }}">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+      Back to CopyPatrol
+    </a>
+    <h1 class="text-center">Leaderboard</h1>
+
+    <div class="text-center clearfix">
+      <span class="leaderboard-column col-sm-4">
```

Comment:
Why are these spans and not div? Aren't spans usually meant for inline things only?